dev/source/docs/porting.rst

```diff
@@ -6,6 +6,12 @@ Porting to a new flight controller board
 
 ArduPilot :ref:`supports a wide variety of flight controllers <common-autopilots>` with new controllers being added all the time.  This page spells out the steps to port ArduPilot to a new board with an emphasis on porting to STM32 based boards (the most common type) using `ChibiOS <http://www.chibios.org/dokuwiki/doku.php>`__.
 
+.. note:: Any firmware customization of ArudPilot code must abid by the terms of the GPL3.0 open source code license. ArduPilot also reminds developers and manufacturers to adhere to the the appropriate trademark and copyright laws when developing new autopilots
```

Comment:
Spelling mistake - `abide`.

Also, I think Tridge was lately asking for GPL3.0+ in some of the repos I recently set up. 

Does this need to end in a period?

Reply:
will add the +...periods on notes are optional....variations both ways throughout...I don't think its an issue